### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by SaadiaELF**
First merge to main

### CODE REVIEW

File: js/index.js [hunk 1]
```diff
@@ -0,0 +1,86 @@
+let startBtn = document.querySelectorAll(".btn--play");
+let pauseBtn = document.querySelectorAll(".btn--pause");
+let stopBtn = document.querySelectorAll(".btn--stop");
+let videos = document.querySelectorAll(".e__card__video");
+
+// Hide menu on mobile view after clicking on menu items
+function hideMenu() {
+  let menuOpen = document.querySelector("#h__side-menu").checked;
+
+  if ((menuOpen = true)) {
```
Comment (DomVinyard) [inline js/index.js]:
```suggestion
  if ((menuOpen === true)) {
```
?

File: js/index.js [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+  // Looping trough all the stop buttons and applying the click event
+  for (let i = 0; i < videos.length; i++) {
+    pauseBtn[i].addEventListener("click", function () {
+      videos[i].contentWindow.postMessage(
+        '{"event":"command","func":"' + "pauseVideo" + '","args":""}',
+        "*"
+      );
+    });
+  }
+}
```
Comment (DomVinyard) [inline js/index.js:47]: will there be multiple videos on the same page?

File: js/index.js [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+
+    let minutes = Math.floor((timeLeft % (1000 * 60 * 60)) / (1000 * 60));
+    let seconds = Math.floor((timeLeft % (1000 * 60)) / 1000);
+    if (seconds < 10) {
+      seconds = "0" + seconds;
+    }
+     if (minutes < 10) {
+       minutes = "0" + minutes;
+     }
+    document.querySelector("#timer").innerHTML = `${minutes}:${seconds}`;
```
Comment (DomVinyard) [inline js/index.js:74]: very nice!

**Review by DomVinyard — APPROVED** [review]
I have approved so you can merge, couple of comments, great work

File: js/index.js (hunk 2 above)
Comment (SaadiaELF) [reply js/index.js:47]: Yes we created a randomise effect, one video is shown at a time from an array of videos

File: js/index.js (hunk 1 above)
Comment (SaadiaELF) [reply js/index.js]: 😅

**Pull request merged**